OPAL/br/src/main/scala/org/opalj/br/fpcf/properties/string_definition/properties.scala

```diff
@@ -0,0 +1,14 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj.br.fpcf.properties
+
+import org.opalj.br.fpcf.properties.string_definition.StringTreeElement
+
+package object properties {
+
+    /**
+     * `StringTree` is used to build trees that represent how a particular string looks and / or how
+     * it can looks like from a pattern point of view (thus be approximated).
+     */
+    type StringTree = StringTreeElement
```

Comment:
why is this needed?

Reply:
I don't think it is, i removed the type alias and renamed the original abstract class to `StringTree` instead (which seems to be more consistent with the naming of all subclasses)